app/src/chainlit.py

```diff
@@ -4,6 +4,8 @@
 from typing import Any
 from urllib.parse import parse_qs, urlparse
 
+from asyncer import asyncify
```

Comment:
Just curious, any particular reason to prefer this over the cl.make_async func?

Reply:
So that the piece of code is not chainlit-specific and the line can later be refactored out of chainlit.py